api/app/Http/Controllers/ChildEntityController.php

```diff
@@ -157,7 +158,7 @@ public function putMany(Request $request, $id)
         $childModels = $this->getChildModel()
             ->hydrateRequestCollection($requestCollection, $existingChildModels);
 
-        $this->getRelation($parent)->saveMany($childModels);
+        $this->saveMany($parent, $childModels);
```

Comment:
@xiphiaz I've started trying out some options here yesterday and thought about this. Will try a bit more regarding the decorator pattern route, and see if I can come up with something that would work.